Source/CesiumRuntime/Private/CesiumGeoJsonObject.cpp

```diff
@@ -0,0 +1,412 @@
+}
+} // namespace
+
+FJsonObjectWrapper UCesiumGeoJsonFeatureBlueprintLibrary::GetProperties(
+    const FCesiumGeoJsonFeature& InFeature) {
+  if (!InFeature._document || !InFeature._feature) {
+    return {};
+  }
+  TSharedPtr<FJsonObject> object = MakeShared<FJsonObject>();
+  if (InFeature._feature->properties) {
+    for (const auto& [k, v] : *InFeature._feature->properties) {
+      object->SetField(UTF8_TO_TCHAR(k.c_str()), jsonValueToUnrealJsonValue(v));
+    }
+  }
+
+  FJsonObjectWrapper wrapper;
+  wrapper.JsonObject = object;
+  return wrapper;
+}
+
+FCesiumGeoJsonObject UCesiumGeoJsonFeatureBlueprintLibrary::GetGeometry(
+    const FCesiumGeoJsonFeature& InFeature) {
+  if (!InFeature._document || !InFeature._feature ||
+      !InFeature._feature->geometry) {
+    return {};
+  }
+
+  return FCesiumGeoJsonObject(
+      InFeature._document,
+      InFeature._feature->geometry.get());
+}
+
+bool UCesiumGeoJsonFeatureBlueprintLibrary::IsValid(
+    const FCesiumGeoJsonFeature& InFeature) {
+  return InFeature._document != nullptr && InFeature._feature != nullptr;
+}
+
+bool UCesiumGeoJsonObjectBlueprintLibrary::IsValid(
+    const FCesiumGeoJsonObject& InObject) {
+  return InObject._pDocument != nullptr && InObject._pObject != nullptr;
+}
+
+ECesiumGeoJsonObjectType UCesiumGeoJsonObjectBlueprintLibrary::GetObjectType(
+    const FCesiumGeoJsonObject& InObject) {
+  if (!InObject._pDocument || !InObject._pObject) {
+    return {};
+  }
+
+  return (ECesiumGeoJsonObjectType)InObject._pObject->getType();
+}
+
+FBox UCesiumGeoJsonObjectBlueprintLibrary::GetBoundingBox(
+    const FCesiumGeoJsonObject& InObject,
+    EHasValue& Branches) {
+  if (!InObject._pDocument || !InObject._pObject) {
+    Branches = EHasValue::NoValue;
+    return {};
+  }
+
+  const std::optional<CesiumGeometry::AxisAlignedBox>& boundingBox =
+      InObject._pObject->getBoundingBox();
+  if (boundingBox) {
+    Branches = EHasValue::HasValue;
+    return FBox(
+        FVector(
+            boundingBox->minimumX,
+            boundingBox->minimumY,
+            boundingBox->minimumZ),
+        FVector(
+            boundingBox->maximumX,
+            boundingBox->maximumY,
+            boundingBox->maximumZ));
+  }
+
+  Branches = EHasValue::NoValue;
+  return {};
+}
+
+FJsonObjectWrapper UCesiumGeoJsonObjectBlueprintLibrary::GetForeignMembers(
+    const FCesiumGeoJsonObject& InObject) {
+  if (!InObject._pDocument || !InObject._pObject) {
+    return {};
+  }
+
+  TSharedPtr<FJsonObject> object = MakeShared<FJsonObject>();
+  for (const auto& [k, v] : InObject._pObject->getForeignMembers()) {
+    object->SetField(UTF8_TO_TCHAR(k.c_str()), jsonValueToUnrealJsonValue(v));
+  }
+
+  FJsonObjectWrapper wrapper;
+  wrapper.JsonObject = object;
+  return wrapper;
+}
+
+namespace {
+FVector degreesToVector(const glm::dvec3& coordinates) {
+  return FVector(coordinates.x, coordinates.y, coordinates.z);
+}
+} // namespace
```

Comment:
Maybe this function used to do more, but at this point I'd suggest just replacing it with `VecMath::createVector`.